components/developer/rust/rustc.p5m

```diff
@@ -34,2532 +34,91 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 # Use OI's default directory for completions
 file etc/bash_completion.d/cargo path=usr/share/bash-completion/completions/cargo
+
+# Manual copy from build/amd64/build/x86_64-sun-solaris/stage2/lib/rustlib/x86_64-sun-solaris/lib/librustc_driver-61711cbc9b978200.so
+link path=usr/lib/$(MACH64)/librustc_driver-61711cbc9b978200.so \
+    target=../rustlib/x86_64-sun-solaris/lib/librustc_driver-61711cbc9b978200.so
+file path=usr/lib/rustlib/x86_64-sun-solaris/lib/librustc_driver-61711cbc9b978200.so
+
+# Use OI's default directory for zsh completions
+file usr/share/zsh/site-functions/_cargo path=usr/share/zsh/vendor-functions/_cargo
+
 file path=usr/bin/cargo
 file path=usr/bin/cargo-clippy
 file path=usr/bin/cargo-fmt
-file path=usr/bin/clippy-driver
-file path=usr/bin/rls
+file path=usr/bin/cargo-miri
+file path=usr/bin/clippy-driver pkg.depend.bypass-generate=.*
+file path=usr/bin/miri pkg.depend.bypass-generate=.*
+file path=usr/bin/rls pkg.depend.bypass-generate=.*
 file path=usr/bin/rust-gdb
+file path=usr/bin/rust-gdbgui
 file path=usr/bin/rust-lldb
-file path=usr/bin/rustc
-file path=usr/bin/rustdoc
+file path=usr/bin/rustc pkg.depend.bypass-generate=.*
```

Comment:
Just out of curiosity, what dependencies are not resolved?

Reply:
libc

Reply:
our system/library or something else?

Reply:
our system/library, I would need to check out in more detail though.